civitai-downloader-v2/src/cli/main.py

```
@@ -412,12 +1170,763 @@ async def run_scan():
+            
+            # Stop the bulk manager
+            await bulk_manager.stop()
+            
+        except Exception as e:
+            click.echo(f"Bulk download failed: {e}", err=True)
+            raise
+    
+    run_async(run_bulk_download())
+
+
+@main.command('history')
```

Ruff (0.12.2)

Undefined name `main`

(F821)